### DISCUSSION TITLE
Rouncube config

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15143] [How to] [answered]
Hello there

I set mail custom server using Iredmail, and after dealing with (possibly) all various configs I still cannot connect. Storage server failed .... Roundcube installed on mails server working perfectly. Is there any recommendation on that? Appreciate help

Best!

**Comment by s2b-git** [comment]
Just to add.. other email clients connecting without issues :-)

**Reply by blu-IT** [reply]
Can you please post a screenshot of your YF / Roundcube config regarding the mail settings?

**Comment by s2b-git** [comment, marked as answer]
resolved dealing with dovecot. Sorry for alarm
